spec/dor/governable_spec.rb

```diff
@@ -11,6 +11,12 @@ class GovernableItem < ActiveFedora::Base
   before(:each) { stub_config   }
   after(:each)  { unstub_config }
```

Comment:
No substantial changes to existing specs (just moved some methods around).  Added one spec and moved others to permissable_spec.